src/Symfony/Component/WebLink/Enum/LinkRelations.php

```diff
@@ -0,0 +1,129 @@
+<?php
+
+namespace Symfony\Component\WebLink\Enum;
+
+enum LinkRelations: string
+{
+    case ABOUT = 'about';
```

Comment:
```suggestion
    case About = 'about';
```

Following https://symfony.com/doc/current/contributing/code/standards.html#naming-conventions